EarlGrey/Matcher/GREYMatchers.m

```diff
@@ -438,9 +438,16 @@ @implementation GREYMatchers
       return NO;
     }
     NSInteger row = [element selectedRowInComponent:column];
-    NSString *rowLabel = [element.delegate pickerView:element
-                                          titleForRow:row
-                                         forComponent:column];
+    NSString *rowLabel;
+    if ([element.delegate respondsToSelector:@selector(pickerView:attributedTitleForRow:forComponent:)]) {
```

Comment:
I would store element.delegate in local var and use that:
id<UIPickerViewDelegate> delegate = element.delegate;

I would also store @selector(...) into a local var.
SEL attributedTitleSel = @selector(pickerView:attributedTitleForRow:forComponent:);
SEL nonAttributedTitleSel = @selector(pickerView:titleForRow:forComponent:);

if ([delegate respondsToSelector:attributedTitleSel]) {
   ...
} else if ([delegate respondsToSelector:nonAttributedTitleSel]) {
   ...
}